packages/forms-web-app/src/dynamic-forms/dynamic-components/radio/index.njk

```diff
@@ -5,6 +5,22 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 
 {% set title = question.pageTitle + " - " + journeyTitle + " - GOV.UK" %}
+
+{% set options = [] %}
+
+{% for option in question.options %}
+  {% set options = (options.push({
+    value: option.value,
+    text: option.text,
+    attributes: { "data-cy": "answer-" + option.value },
+    checked: answer == option.value,      
+    conditional: {
+      html: option.conditional.html,
+      value: option.conditional.fieldName
+    }
+  }), options) %}
+{% endfor %}
```

Comment:
I'm wondering whether we need this logic in the template to define the options array. As far as I can see, `question.options` is almost identical, with items containing all of those fields except for `attributes` which could be added in the `prepQuestionForRendering` method of the options question as we do for the `checked` attribute.